carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/utils/mobile/notifications/NotificationUtils.java

```diff
@@ -21,9 +21,10 @@
 
 import com.google.gson.JsonObject;
 import com.google.gson.JsonParser;
-import com.jayway.restassured.response.Response;
 import com.qaprosoft.carina.core.foundation.utils.rest.RestUtil;
 
+import io.restassured.response.Response;
+
 public class NotificationUtils {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://demo.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS1118)